workflows/pipe-common/scripts/autoscale_sge.py

```
@@ -1415,6 +1482,7 @@ def fetch_worker_launch_system_params(api, master_run_id):
                                          storage_file=os.path.join(shared_work_dir, '.autoscaler.storage'))
     scale_up_timeout = int(api.retrieve_preference('ge.autoscaling.scale.up.timeout', default_value=30))
     scale_down_timeout = int(api.retrieve_preference('ge.autoscaling.scale.down.timeout', default_value=30))
+    idle_timeout = get_idle_timeout()
```

I believe it's okay to fail fast here in case specified value is invalid. Also it is a nice feature and I believe we can use it by default with some fixed value like 30 seconds.

```
idle_timeout = int(os.getenv('CP_CAP_AUTOSCALE_IDLE_TIMEOUT', 30))
```

As a result we can handle this value as non null everywhere.

---

Done